### DISCUSSION TITLE
Regarding the HingeJoint presence issue

### DISCUSSION THREAD

**Discussion by zhy3221** [cyberbotics/webots#6314] [General]
Hello, in webots, HingeJoint adds the motorof the device to control endpoint, why does position control of the motor cause  the parent point to make a small movement?

**Comment by omichel** [comment]
This is simply due to the laws of physics.
If you do the same on a real system, you will observe the same movement.
The bigger the mass of the parent, the smaller the motion.

*Note*: if the parent is the static world, then obviously no motion will occur at the parent.

**Reply by zhy3221** [reply]
Thank you very much for your reply, I can understand the reason, but for robotic arm kinematics control then each arm is movable, how to design the webot so that the HingeJoint doesn't affect the parent point's arm.

**Reply by omichel** [reply]
The base of the robot should be either heavy enough or linked to the static environment.

**Reply by zhy3221** [reply]
For example, for the two arms L-1 and L connected by the robotic arm joint K, how can the control of the joint K enable the movement of only L, while the movement of L-1 is only the effect produced by the joint K-1.

**Reply by omichel** [reply]
The joint K-1 could be attached to the static environment.
